src/Command/RunCommand.php

```diff
@@ -59,6 +60,7 @@ public function __construct(
         private readonly string $jobId,
         private readonly string $storageApiToken,
         private readonly array $instanceLimits,
+        private readonly ?LoggerInterface $debugLogger = null,
```

Comment:
🤦 Mi chvili trvalo nez jsem to pochopil.
Prvni $logger je monolog logger, tohle by melo pridat ten interni symfoni. Ten nebude mit to hadnlovani do storage event a melo by byt ok.

Nemelo by to pak ale byt spis `Symfony\Component\HttpKernel\Log\Logger`? Protoze i Monolog implementuje interface `Psr\Log\LoggerInterface`

Docela se divim ze to jde a nezarve to ze nevi jakou service tam ma predat.

Reply:
Symfony magic https://symfony.com/doc/current/logging/channels_handlers.html#how-to-autowire-logger-channels

Kdyby to bylo neco dlouhodobyho, asi bych to preventivne nawiroval rucne, ale jelikoz je to jen docasny, byl jsem linej.